Expensify/App
# Add more logs to help debug random sign outs

### Details
While looking into the linked issue I noticed we are not logging when we fail to reauthenticate so added some JS logs for that. I doubt we are explicitly calling `signOut()` but also added a log for that for my own sanity.

### Fixed Issues
Fixes https://github.com/Expensify/Expensify/issues/152639

### Tests
1. Sign in and sign out
2. Verify the logs appear
<img width="872" alt="Screen Shot 2021-01-28 at 9 26 41 AM" src="https://user-images.githubusercontent.com/32969087/106188745-6cc19700-614b-11eb-9891-11eb23b1e83b.png">

3. Invalidate both the `authToken` and `autoGeneratedLogin` directly in `localStorage` by editing their values
4. Tap on a new chat
5. Verify logs appear

<img width="869" alt="Screen Shot 2021-01-28 at 9 29 54 AM" src="https://user-images.githubusercontent.com/32969087/106188882-9f6b8f80-614b-11eb-9f1d-d328501630d9.png">

### Tested On


### Screenshots


#### Web


#### Mobile Web


#### Desktop


#### iOS


#### Android
